config/assets/default.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
         'public/lib/angular-animate/angular-animate.js',
         'public/lib/angular-bootstrap/ui-bootstrap-tpls.js',
         'public/lib/ng-file-upload/ng-file-upload.js',
-        'public/lib/ng-img-crop/compile/unminified/ng-img-crop.js',
+        'public/lib/ng-img-crop-full-extended/compile/minified/ng-img-crop.js',
```

Comment:
Better include unminified version, easier for debugging and our build process should do the minification instead.